compiler/one-cmds/one-codegen

```diff
@@ -94,15 +95,40 @@ def _get_parser(backends_list):
     return parser
 
 
-def _verify_arg(parser, args):
+def _verify_arg(parser, args, cfg_args):
     """verify given arguments"""
+    cmd_backend_exist = oneutils.is_valid_attr(args, 'backend')
+    cfg_backend_exist = oneutils.is_valid_attr(cfg_args, 'backend')
+    cfg_backends_exist = oneutils.is_valid_attr(cfg_args, 'backends')
+    if cfg_backend_exist and cfg_backends_exist and oneutils.is_valid_attr(
+            cfg_args, 'command'):
```

Comment:
Maybe this would be correct? IIUC, this if statement checks the use of `backend` and `backends` in cfg file, without consideration of command line.

```suggestion
    if cfg_backend_exist and cfg_backends_exist:
```

compiler/one-cmds/tests/onecc_neg_028.cfg may be related to this change.

Reply:
Well, both seems correct because `backend` and `command` are kinda a set. But I think your suggestion covers broader negative cases.

Reply:
Ah, I misunderstood the code. This is about `command` in cfg file, not about `cmd_backend_exist`.. Sorry for confusion.